docs/usage/key-concepts/pull-requests.md

```diff
@@ -76,6 +76,7 @@ If you regularly wish to close immortal PRs, it's an indication that you may be
 ### How to fix immortal PRs
 
 Avoid grouping dependencies together which have different versions, or which you have a high chance of wanting to ignore.
+If you have immortal PRs which you want to keep closed, then set `"recreateWhen": "never"`.
```

Comment:
It depends whether you want it applied to every PR or not. There's certainly a case where people may decide they never want immortal PRs recreated

Reply:
Maybe we need text for these cases then?

- Ignore a immortal update for a certain package(s)
- Ignore all immortal PRs

Reply:
I don't think it's necessary, because "you can apply settings across the repo or via package rules" is a common concept and we don't want to repeat it for every config option we have.